app/src/main/java/com/jerboa/ui/theme/Theme.kt

```diff
@@ -55,7 +55,7 @@ fun JerboaTheme(
                 ),
             )
         } else {
-            pink()
+            blue()
```

Comment:
I don't like the pink theme as default 👯

Reply:
Haha alright.